Improve doc for RBAC group task (#4287)
istio/istio.io: content/docs/tasks/security/rbac-groups/index.md

    EOF
    {{< /text >}}

>     Policy needs some time to take affect. Wait at least one minute.

Reviewer: ```suggestion
    Policies take effect after some time, please wait at least one minute.
```

Author: Done. Thanks!

    EOF
    {{< /text >}}

>     Policies take effect after some time, please wait at least one minute.

Reviewer 2: Is there a way to check when the policies have taken effect? If so, document it, and update all the other places in this doc that request the user to wait for policies to take effect.

1.  Enable the Istio RBAC for the namespace:

    {{< warning >}}

>     This is a cluster-wide policy. If your cluster is using another `ClusterRbacConfig`, it will be overwritten.

Reviewer: ```suggestion
    This is a cluster-wide policy and Istio overwrites any another `ClusterRbacConfig`.
```

Author: Done. Thanks!

1.  Enable the Istio RBAC for the namespace:

    {{< warning >}}

>     This is a cluster-wide policy and Istio overwrites any another `ClusterRbacConfig`.

Reviewer: Please include the nouns. For example `This` without a noun might lead to readers not knowing what you are referencing. Consider:
```suggestion
    The Istio RBAC enablement is a cluster-wide policy and Istio overwrites any previously set `ClusterRbacConfig` configuration.
```

[python script]({{<github_file>}}/security/tools/jwt/samples/gen-jwt.py)
to generate a JWT with other list-typed claims for testing purposes.
Follow the instructions in the `gen-jwt` script to use the `gen-jwt.py` file.

> For the `key.pem` file, you may use one at

Reviewer: ```suggestion
To get the `key.pem` file, run the following command.
```

Author: Done. Thanks!

[python script]({{<github_file>}}/security/tools/jwt/samples/gen-jwt.py)
to generate a JWT with other list-typed claims for testing purposes.
Follow the instructions in the `gen-jwt` script to use the `gen-jwt.py` file.

> To get the `key.pem` file, run the following command:

Reviewer: Please make this part of the numbered steps.
```suggestion
1. Get the `key.pem` file with the following command:
```
```suggestion
To get the `key.pem` file, run the following command:
```

> {{< text bash >}}

Reviewer: Please indent this code block with 4 spaces to ensure the numbered list doesn't break.

> $ wget {{< github_file >}}/security/tools/jwt/samples/key.pem

Reviewer 2: Replace `wget` with `curl`. The majority of the docs use `curl`:
```
$ grep -r wget content/ | wc -l
       4
$ grep -r curl content/ | wc -l
     263
```
